apps/E2E/src/Button/pages/ButtonPageObject.ts

```diff
@@ -63,6 +63,10 @@ class ButtonPageObject extends BasePage {
   get _pageButton() {
     return By(HOMEPAGE_BUTTON_BUTTON);
```

Comment:
@samuelfreiberg I propose updating the _pageButton function to use the _pageButtonName in it.
`  get _pageButton() {
    return By(this._pageButtonName);
  }`

Reply:
Similar thing for the _testPage function.
We could have the _testPage in the BasePage itself set to 
` get _testPage() {
    return By(this._pageName);
  }`
This would reduce redundancy.

Reply:
I think this is a good change!

Reply:
Thanks for confirming, Will take this up in a separate PR!